docs/manual/article-management/insert-tags.de.md

```diff
@@ -199,3 +199,16 @@ Verfügbare Flags:
 | `readable_size`     | Wandelt die Ausgabe in ein menschenlesbares Format um.                   | siehe `System::getReadableSize()`                 |
 | `urlencode`         | URL-kodiert einen String.                                                | [PHP-Funktion](https://php.net/urlencode)         |
 | `rawurlencode`      | URL-Kodierung nach RFC 3986.                                             | [PHP-Funktion](https://php.net/rawurlencode)      |
+
+
+## Basic Entities
+
+Folgende »Basic Enities« werden von Contao in die jeweiligen HTML Entities zurück umgewandelt:
+
+| Basic Entities | HTML Entities |
+|:---------------|:--------------|
+| `[&]`          | `&amp;`       |
+| `[lt]`         | `&lt;`        |
+| `[gt]`         | `&gt;`        |
+| `[nbsp]`       | `&nbsp;`      |
+| `[-]`          | `&shy;`       |
```

Comment:
Ich finde man könnte hier noch ein wenig die use cases beschreiben :). z.B. können diese Basic Entities ja überall verwendet werden, wie Insert Tags - und daher auch bspw. in Headlines. Das ist bspw. bei langen, zusammengsetzten Wörtern im Deutschen wichtig, damit man dann

```
Donau[-]dampf[-]schiff[-]fahrts[-]gesell[-]schaft
```

vom Browser automatisch abgekürzt werden kann.

Für die anderen weiß ich eigentlich keine guten Use-Cases ;P